.github/workflows/flake8.yml

```diff
@@ -0,0 +1,25 @@
+name: flake8
+
+on:
+  pull_request:
+    paths:
+      - "tools/*.py"
+      - ".github/workflows/flake8.yml"
+  push:
+    branches:
+      - '**'
+    paths:
+      - "tools/*.py"
+      - ".github/workflows/flake8.yml"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Lint with flake8
+        uses: py-actions/flake8@v2
```

Comment:
Where is configured which files are passed to flake8? Does this correspond to the paths above? But I guess these are only for the trigger, right?

Reply:
We could specify a [path](https://github.com/py-actions/flake8#path) (`tools/` I guess) for flake8, but maybe it's not even necessary.

The paths above are just to specify that this CI workflow has to be run on changes that touch those paths

Reply:
We would know if we need to set the path, if we could see the CI running in this PR :wink: